luigi/server.py

```diff
@@ -14,10 +14,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-"""
-Simple REST server that takes commands in a JSON payload.
-Interface to the :py:class:`~luigi.scheduler.CentralPlannerScheduler` class.
-"""
```

Comment:
Interesting. I didn't touch anything about those lines but something happened. I will change them to the original status!